From 37350151e311c624b35da5733eba05c3bf6fdad9 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Mon, 10 Feb 2020 16:02:06 +0200
Subject: [PATCH] [SW-0] gaudi: pll: remove hl_pll_is_configurable()

Since the MF Gaudi chip the CPU PLL configuration has been verified to
work correctly during preboot (executing on ARM CPU).
Meaning, we can configure the CPU PLL which was skipped till now.
Therefore, there is no more need for the Gaudi private implementation of
hl_pll_is_configurable() function (which only skipped the CPU PLL).
We still do not add the CPU PLL configuration to preboot, just remove
the protection.

Change-Id: I5420346e7ebe924921d740c8a42204388e92e841
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/gaudi/asic.c b/gaudi/asic.c
index a15bd63..ffbc087 100644
--- a/gaudi/asic.c
+++ b/gaudi/asic.c
@@ -156,18 +156,6 @@
 	},
 };
 
-bool hl_pll_is_configurable(struct hl_pll_cfg *cfg)
-{
-	switch (cfg->base) {
-	case mmPSOC_CPU_PLL_BASE:
-		return false;
-	default:
-		break;
-	}
-
-	return true;
-}
-
 int hl_pll_get_cfg(struct hl_pll_cfg **hl_pll_cfg)
 {
 	int pll_num = ARRAY_SIZE(gaudi_pll_cfg);
